pkgs/development/python-modules/numpy-groupies/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  numpy,
+  numba,
+  pandas,
+}:
+
+buildPythonPackage rec {
+  pname = "numpy-groupies";
+  version = "0.11.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ml31415";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Eu+5SR28jIasKe1p7rvbq2yo3PGZRQWWdG3A5vGhnyM=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    numba
+    pandas
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/ml31415/numpy-groupies";
```

```suggestion
    homepage = "https://github.com/ml31415/numpy-groupies";
    changelog = "https://github.com/ml31415/numpy-groupies/releases/tag/v${version}";
```
Adding `meta.changelog` is useful for future updates.